.drone.yml

```diff
@@ -19,7 +19,7 @@ steps:
     settings:
       dry_run: true
       build_args:
-        - SERVER_VERSION=ltsc2019
+        - SERVER_VERSION=1809
```

Comment:
why are we reverting to 1809?

Reply:
Good catch, my IDE keeps auto formatting this file incorrectly. I must have copy+pasted the old format and forgot to update this line